src/Milky/Checker.cs

```diff
@@ -230,17 +230,40 @@ private void OutputCombo(Combo combo, CheckResult checkResult)
                     Directory.CreateDirectory(Path.GetDirectoryName(outputPath));
 
                     File.AppendAllText(outputPath, outputString + Environment.NewLine);
-                }
 
-                Console.ForegroundColor = checkResult.ComboResult switch
-                {
-                    ComboResult.Hit => _outputSettings.HitColor,
-                    ComboResult.Free => _outputSettings.FreeColor,
-                    ComboResult.Invalid => _outputSettings.InvalidColor
-                };
+                    if (_outputSettings.GlobalOutput)
+                    {
+                        string globalOutputPath = Path.Combine(Path.Combine(_outputSettings.OutputDirectory != null ? Directory.GetParent(_outputSettings.OutputDirectory).FullName : "Results", "Global"), outputFile + ".txt");
```

Comment:
Please reverse this condition (replace `!= null` by `== null` and edit the code accordingly) - That's just so it follows the current codebase pattern.